src/components/Input.tsx

```diff
@@ -68,23 +70,26 @@ export const FileInput: React.FC<FileInputProps> = ({ options, onFileSelect, ...
     );
 };
 
-export const SliderInput: React.FC<SliderProps> = ({ ...props }) => (
-    <Slider {...props} />
-);
+export const SliderInput: React.FC<SliderProps> = ({ ...props }) => <Slider {...props} />;
 
-export const ToggleInput: React.FC<InputHTMLAttributes<HTMLInputElement>> = ({ className, ...props }) => (
-    <div className="relative w-10 h-5">
-        <input
-            type="checkbox"
-            className={classNames('opacity-0 w-full h-full cursor-pointer peer', className)}
-            {...props}
-        />
-        <div className="absolute top-0 left-0 w-full h-full rounded-full bg-silver-700 pointer-events-none" />
-        <div
-            className={classNames(
-                'absolute top-1 left-1 w-3 h-3 rounded-full bg-silver-600 duration-100 pointer-events-none',
-                'peer-checked:bg-amethyst-400 peer-checked:left-6',
-            )}
-        />
-    </div>
+interface ToggleInputProps extends InputHTMLAttributes<HTMLInputElement> {
+    tooltip: string;
+}
+export const ToggleInput: React.FC<ToggleInputProps> = ({ tooltip, className, ...props }) => (
+    <Tippy content={tooltip} appendTo={document.body}>
+        <div className="relative w-10 h-5">
+            <input
+                type="checkbox"
+                className={classNames('opacity-0 w-full h-full cursor-pointer peer', className)}
+                {...props}
+            />
+            <div className="absolute top-0 left-0 w-full h-full rounded-full bg-silver-700 pointer-events-none" />
+            <div
+                className={classNames(
+                    'absolute top-1 left-1 w-3 h-3 rounded-full bg-silver-600 duration-100 pointer-events-none',
+                    'peer-checked:bg-amethyst-400 peer-checked:left-6',
+                )}
+            />
+        </div>
+    </Tippy>
```

Comment:
Let's keep it simple and just define Tooltips around the component when necessary. I'd just leave the `ToggleInput` as it was and throw the Tippy stuff into `instrument.tsx` directly.

Reply:
This doesn't work for some reason
```tsx
<Tippy content="Toggle Auto-Refresh" appendTo={document.body}>
    <ToggleInput onClick={handleWatch} />
</Tippy>
```

Reply:
Might need to forward ref on the toggle input component.